tutorials/xr/a_better_xr_start_script.rst

```diff
@@ -91,7 +91,16 @@ We introduce a few new variables to our script as well:
 Our updated ready function
 --------------------------
 
-The ready function mostly remains the same but we hook up a number of signals that will be emitted by the :ref:`XRInterface <class_xrinterface>`.
+We add a few things to the ready function.
+
+If we're using the mobile or forward+ renderer we set the viewports ``vrs_mode`` to ``VRS_XR``.
+On platforms that support this, this will enable foveated rendering.
+
+If we're using the compatibility renderer, we check if the OpenXR foveated rendering settings
+are configured and if not, we output a warning.
```

Comment:
This part is a bit confusing to me. The previous paragraph says to only change settings for Mobile and Forward+, but we put in a warning for when we intentionally don't do anything for the compatibility renderer?

Also while I understand the point of having the warning in the code for the XRTools addon, I'm not sure it makes sense to include it here where users would be the ones typing out the code. Especially since users would have just enabled the setting.

Reply:
The idea here is that we include this script in every XR entry on godot-demo-projects and I'm trying to keep the script the same over all projects. This help page just further explains each part of that script.

So you are right, someone who uses this help page to create their own version of this script, would skip over this and just configure things the way they want depending on their choice of renderer.

But those who grab the script as is from the demo projects to include in their own projects, for them it's a little extra to nudge them to enable foveated rendering as they may not realise there are separate settings for compatibility.